content/docs/add-react-to-a-website.md

```diff
@@ -170,7 +170,7 @@ Adding JSX to a project doesn't require complicated tools like a bundler or a de
 Go to your project folder in the terminal, and paste these two commands:
 
 1. **Step 1:** Run `npm init -y` (if it fails, [here's a fix](https://gist.github.com/gaearon/246f6380610e262f8a648e3e51cad40d))
-2. **Step 2:** Run `npm install babel-cli@6 babel-preset-react-app@3`
```

Comment:
Had troubles of using the latest version of `babel-preset-react-app`, mostly because the default compiled JS to ESM so users would have to add `script type="module"` and run index.html with local server like `serve`. It's just easier just to use `@babel/preset-react` instead and there's no issues.

Reply:
Hmm, it was using the `app` preset was very intentional because it has a bunch of other plugins. I think there's a config option for the preset that disables the ESM?